hathor/websocket/factory.py

```diff
@@ -86,16 +89,24 @@ def __init__(self, metrics: Optional[Metrics] = None, wallet_index: Optional[Wal
 
         self.is_running = False
 
+        self.log = logger.new()
+
+        # A timer to periodically broadcast dashboard metrics
+        self._lc_send_metrics = LoopingCall(self._send_metrics)
+        self._lc_send_metrics.clock = reactor
+
     def start(self):
         self.is_running = True
 
         # Start limiter
         self._setup_rate_limit()
 
         # Start metric sender
-        self._schedule_and_send_metric()
+        self._lc_send_metrics.start(1, now=False)
```

Comment:
Move `1` to a constant (e.g. `LC_SEND_METRICS_INTERVAL`).

Reply:
Done.